crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -233,6 +233,11 @@ impl EventTimelineItem {
         matches!(self.kind, EventTimelineItemKind::Remote(_))
     }
 
+    /// Check whether this item's content is UTD (unable to decrypt).
+    pub fn is_utd(&self) -> bool {
```

Comment:
This method is on the wrong type; but I've seen that it's got already merged in a separate PR, so I've moved another PR to move it out elsewhere: https://github.com/matrix-org/matrix-rust-sdk/pull/4782